react/features/toolbox/components/Toolbox.web.js

```diff
@@ -355,12 +355,12 @@ class Toolbox extends Component<Props, State> {
                         </div> }
                 </div>
                 <div className = 'button-group-center'>
-                    { this._shouldShowButton('microphone')
-                        && <AudioMuteButton /> }
-                    { this._shouldShowButton('hangup')
-                        && <HangupButton /> }
-                    { this._shouldShowButton('camera')
-                        && <VideoMuteButton /> }
+                    <AudioMuteButton
+                        visible = { this._shouldShowButton('microphone') } />
```

Comment:
After debating this with myself for a while I chose to make `visible` a prop. The idea is that the button is a pretty-dumb entity really, all it knows is how to paint itself and what action to perform, it's the responsibility of the container to decide if that button should be there or not.

I'm wide open to other ideas regarding this.